src/Microsoft.PowerShell.Commands.Utility/commands/utility/WebCmdlet/Common/WebRequestPSCmdlet.Common.cs

```diff
@@ -1707,9 +1707,7 @@ private void AddMultipartContent(object fieldName, object fieldValue, MultipartF
         private static StringContent GetMultipartStringContent(object fieldName, object fieldValue)
         {
             ContentDispositionHeaderValue contentDisposition = new("form-data");
-
-            // .NET does not enclose field names in quotes, however, modern browsers and curl do.
-            contentDisposition.Name = "\"" + LanguagePrimitives.ConvertTo<string>(fieldName) + "\"";
+            contentDisposition.Name = LanguagePrimitives.ConvertTo<string>(fieldName);
```

Comment:
This broke IWR for us when we upgraded to 7.4.  We use IWR in a publish script to upload a zip to an internal PERL / CGI-based website.  AFAICT the PERL CGI module requires field name values to be quoted.